code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -12,6 +12,7 @@
 	force = 5
 	var/smash_duration = 5 //Directly relates to the 'weaken' duration. Lowered by armor (i.e. helmets)
 	var/isGlass = 1 //Whether the 'bottle' is made of glass or not so that milk cartons dont shatter when someone gets hit by it
+	matter = list("glass" = 800)

```

Comment:
Small math inconsistency. Glass holds 30 volume and is 300, hence it is 10 glass per 1 volume. But this holds 100 and cost is 800 which means it is 8 glass per 1 volume

Meaning it should be 1000

Reply:
The beaker and glass are also inconsistent between each other. You're free to headcanon that the drinking glass has a thicker surface. I think the cost is fine.

Reply:
Aren't bottles plastic?

Reply:
Regular bottles are glass, as they can leave glass shards when you throw, I think.